Cubical/Data/FinSet/Properties.agda

```diff
@@ -0,0 +1,151 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Data.FinSet.Properties where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Structure
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Equiv
+
+open import Cubical.HITs.PropositionalTruncation
+
+open import Cubical.Data.Nat
+open import Cubical.Data.Unit
+open import Cubical.Data.Empty renaming (rec to EmptyRec)
+open import Cubical.Data.Sigma
+
+open import Cubical.Data.Fin
+open import Cubical.Data.SumFin renaming (Fin to SumFin) hiding (discreteFin)
+open import Cubical.Data.FinSet.Base
+
+open import Cubical.Relation.Nullary
+open import Cubical.Relation.Nullary.DecidableEq
+open import Cubical.Relation.Nullary.HLevels
+
+private
+  variable
+    ℓ ℓ' : Level
+    A : Type ℓ
+
+-- infix operator to more conveniently compose equivalences
+
+_⋆_ = compEquiv
+
+infixr 30 _⋆_
+
+-- useful implications
+
+isFinSetFin : ∀ {n} → isFinSet (Fin n)
+isFinSetFin = ∣ _ , pathToEquiv refl ∣
+
+isFinSetUnit : isFinSet Unit
+isFinSetUnit = ∣ 1 , Unit≃Fin1 ∣
+
+isFinSet→Discrete : {X : Type ℓ} → isFinSet X → Discrete X
```

Comment:
Can't you replace `X` by `A` here as you already have `A` as a variable? The same comment applies to a few other results in this file